app/assets/stylesheets/views/_landing_page/grid.scss

```diff
@@ -2,8 +2,8 @@
 
 @include govuk-media-query($from: desktop) {
   .grid-container {
-    display: grid;
-    grid-template-columns: repeat(3, 1fr);
+    display: flex;
+    justify-content: space-between;
```

Comment:
Curious why this changed from grid to flex? It works fine so I'm happy to approve, just curious.

Reply:
Good question. I initially changed it as part of investigating the problem with chart resizing. When I realised that it was equivalent I thought it might be better to keep it all as flexbox, just to avoid mixing the two.

Reply:
Thanks for the review 👍